charts/prometheus/README.md

```diff
@@ -6,7 +6,8 @@ This chart bootstraps a [Prometheus](https://prometheus.io/) deployment on a [Ku
 
 ## Prerequisites
 
-- Kubernetes 1.3+ with Beta APIs enabled
+- Kubernetes 1.16+
```

Comment:
If we go for kubernetes 1.16 all extension beta apis should be removed.

For example: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/templates/server/ingress.yaml#L10